src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="sorry we dont have any images now"
+          class="logo"
+        />
+      </a>
+      <nav class="roboto-medium">
+        <ul class="nav__list">
+          <a
+            class="page-link is-active"
+            href="#"
+          >
+            <li>Apple</li>
+          </a>
```

Comment:
```suggestion
          <li
            class="nav__item"
          >
             <a href="#" class="nav__link  is-active">Apple</a>
          </li>
```
tag `a` should be inside the `li`